styles/styles.css

```diff
@@ -433,6 +433,15 @@ button {
   margin: 0;
 }
 
+main .leadership-content img {
+  border-radius: 15px;
+}
+
+main .leadership-head img {
```

Comment:
`leadership-head` section should be replaced by `leadspace` block once it is available so this custom styling can be removed eventually. Only the `leadership-content` overrides on top of the `column` block should be needed to build the Leadership page.